_examples/nodeconfigs/simple.json

```diff
@@ -0,0 +1,20 @@
+{
+  "logger": {
+    "color": true,
+    "level": "info"
+  },
+  "nexus_name": "mynexus",
+  "tags": {
+    "foo": "bar"
+  },
+  "workload_types": [
+    {
+      "name": "CONFIGWORKLOAD",
+      "agenturi": "nats://uri/config",
```

Comment:
to work by default, this likely needs to be empty so the node can create a default from the resource directory + name. We can document using URIs in a different config.

Reply:
🤔